src/run/uploader/upload.rs

```diff
@@ -48,9 +49,11 @@ async fn retrieve_upload_data(
                     .map(|body| body.error)
                     .unwrap_or(text);
                 bail!(
-                    "Failed to retrieve upload data: {}\n{}",
+                    "Failed to retrieve upload data: {}\n  -> {} {}",
                     status,
-                    error_message
+                    style("Reason:").bold(),
+                    // !!! we have to manually apply `.red()` this because the `.red()` from the logger is not applied to everything that follows the first `style()` call, for some reasons 🤷‍♂️🤷‍♂️🤷‍♂️
```

Comment:
This might be because of the termination code?
https://gist.github.com/fnky/458719343aabd01cfb17a3a4f7296797#colors--graphics-mode
It resets everything not just the local style. Maybe that can explain the issue.

Reply:
Nice catch!
As seen together, I will change the comment and we will likely refactor the terminal coloring later on